src/common/events/CMakeLists.txt

```diff
@@ -36,6 +36,11 @@ add_library(
   ${EVENT_SOURCES}
 )
 
+target_link_libraries(mirevents
+  LINK_PUBLIC
```

Comment:
Is there a reason we're using `LINK_PUBLIC` rather than just `PUBLIC`? [`LINK_PUBLIC`](https://cmake.org/cmake/help/v3.16/command/target_link_libraries.html#libraries-for-a-target-and-or-its-dependents-legacy) is described as legacy.